src/utils/src/data-container-interface.ts

```diff
@@ -60,6 +60,21 @@ export interface DataContainerInterface {
    */
   column(columnIndex: number): Generator<any, void, any>;
 
+  /**
+   * Returns the column object at the specified index.
+   * @param columnIndex Column index.
+   * @returns The column object at the specified index.
+   */
+  getColumn?(columnIndex: number): any;
```

Comment:
Avoid return type `any` if at all possible. Usually preferable to use `unknown` and force the app to state what type it is expecting via a cast.